### ISSUE TITLE
Cannot install legit through npm

### ISSUE THREAD

**Issue by ridhahassoun** [captainsafia/legit#18]
Running `npm install --global @captainsafia/legit` fails after running postinstall commands `mkdir licenses && node load-licenses.js`. 

```
/usr/local/bin/legit -> /usr/local/lib/node_modules/@captainsafia/legit/index.js

> @captainsafia/legit@2.0.0 postinstall /usr/local/lib/node_modules/@captainsafia/legit
> mkdir licenses && node load-licenses.js

mkdir: licenses: File exists
```
Not sure why it would exist (I checked, it doesn't.)

Attached is the debug log:
[2017-03-04T08_48_57_928Z-debug.txt](https://github.com/captainsafia/legit/files/818495/2017-03-04T08_48_57_928Z-debug.txt)

**Comment by Ma27** [comment]
I can confirm this.

Not sure, but this package might have been published without blacklisting the `licenses/` directory in the `.npmignore`.

**Comment by captainsafia** [comment]
Whoops! Guess I shouldn't have rushed to make this release before a dinner meeting, huh? 😜 

Fixed in  cf77482.
